src/sections/Home/Banner/index.js

```diff
@@ -1,24 +1,25 @@
 import React, { useEffect, useState } from "react";
 import BannerDefault from "../Banner-default";
-// import BannerAlt from "../Banner-alt"; 
+// import BannerAlt from "../Banner-alt";
 
 const Banners = [
   <BannerDefault key={0} />,
-  // <BannerAlt key={1}/> 
+  // <BannerAlt key={1} /> 
 ];
 
 const Banner = () => {
```

Comment:
rename: RandomBanner?

Reply:
renaming component from Banner to RandomBanner?

Reply:
Yeah